3rd-party/extensions/hotkeys/js/jquery.fancytree.hotkeys.js

```diff
@@ -15,9 +15,11 @@
 	var initHotkeys = function(tree, data) {
 		$.each(data, function(event, keys) {
 			$.each(keys, function(key, handler) {
-				$(tree.$container).on(event, null, key, function() {
+				$(tree.$container).on(event, null, key, function(evt) {
 					var node = tree.getActiveNode();
 					handler(node);
+					evt.preventDefault();
```

Comment:
Wouldn't it be more flexible to let the handler *decide* whether he wants to prevent default handling, stop propagation or both?
What do you think about passing `evt` to the handler (which may be useful in itself), The handler can then inspect the event or call methods on it.
The main use case (stopping default handling) could be made easier to implement if we return the handler's response:
```js
return handler(node);
```
This would allow handlers to simply return `false`.

If you feel that this is too much burden on the handlers, then I suggest that we make the 'prevent default' behavior configurable in the plugin options.